substrate/frame/multisig/src/migrations/v2/weights.rs

```diff
@@ -0,0 +1,84 @@
+// This file is part of Substrate.
```

Comment:
did not touch this file at all, don't know how to generate weights for this purpose

Reply:
Yea we can re-generate them as last step.